pkgs/development/python-modules/bleak/default.nix

```diff
@@ -19,7 +21,7 @@ buildPythonPackage rec {
       --replace \"bluetoothctl\" \"${bluez}/bin/bluetoothctl\"
   '';
 
-  propagatedBuildInputs = [ txdbus ];
+  propagatedBuildInputs = [ dbus-next ];
   checkInputs = [ pytest pytestcov ];
 
   checkPhase = "AGENT_OS=linux py.test";
```

Comment:
This should be converted to pytestCheckHook.

Reply:
I have updated the check inputs. Thanks.